Makefile

```
@@ -65,6 +65,16 @@ get-chart: ## Get the chart package and prov file from its repository
 delete-chart: ## Delete the chart package and prov file from its repository
 	rm -f ./charts/$(CHART_NAME)-$(CHART_VERSION).tgz*
 
+.PHONY: validate-chart
+validate-chart: $(HELM) ## Validate a specific chart package
```

checkmake (0.3.2)

[warning] Target body for "validate-chart" exceeds allowed length of 5 lines (7).

(maxbodylength)